src/Initializer/time_stepping/LtsWeights.cpp

```diff
@@ -118,7 +118,7 @@ int seissol::initializers::time_stepping::LtsWeights::getBoundaryCondition( int
                                                                             unsigned cell,
                                                                             unsigned face ) {
   int bcCurrentFace = ((boundaryCond[cell] >> (face*8)) & 0xFF);
-  if (bcCurrentFace > 6) {
+  if (bcCurrentFace > 7) {
```

Comment:
I think you had 8 there in PUMLReader?? Maybe one should put this logic in a separate function, maybe as part of FaceType